ot/lp/__init__.py

```diff
@@ -21,6 +21,8 @@
 
 __all__=['emd', 'emd2', 'barycenter', 'free_support_barycenter', 'cvx']
 
+__all__=['emd', 'emd2', 'barycenter', 'cvx']
```

Comment:
delete this line, **free_support_barycenter** is a new function that has to be imported